lib/sift/client.rb

```diff
@@ -20,12 +21,17 @@ class Response
     #   a JSON object that can be decoded into status, message and request
     #   sections.
     #
-    def initialize(http_response, http_response_code)
-      @body = MultiJson.load(http_response)
-      @request = MultiJson.load(@body["request"].to_s) if @body["request"]
+    def initialize(http_response, http_headers, http_response_code, http_response_class)
       @http_status_code = http_response_code
-      @api_status = @body["status"].to_i
-      @api_error_message = @body["error_message"].to_s
+      @http_class = http_response_class
```

Comment:
No my bad, if they actually are classes and not instances then your naming sounds right